### PR TITLE
Fixes #9375: Add documentation on new API to start remote agent run

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/9375

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Close, do not merge now, it does not build

**Comment by VinceMacBuche** [comment]
Fixed!

**Comment by VinceMacBuche** [comment]
Commit modified

File: src/api_v2/node.md [hunk 1]
```diff
@@ -595,6 +595,109 @@ HTTP/1.1 200 OK
...
+
+    /**
+    @api {post} /api/nodes/{id} 7. Trigger all agent run
+    @apiVersion 8.0.0
+    @apiName applyPolicyAllNodes 
+    @apiGroup Nodes
+
+    @apiParam (Multi) {class}       classes Set class passed as parameter to make it available during agent run.
+
+    @apiDescription This API allows to trigger an agent run on the target node. Response content a json stating if agent has started on each node.
```
Comment (fanf) [inline src/api_v2/node.md]: You need to be more precise on the behavior of the api regarding node answer: what exactly the api is waiting for to build the json answer ?

File: src/api_v2/node.md [hunk 2]
```diff
@@ -595,6 +595,109 @@ HTTP/1.1 200 OK
...
+    /**
+    @api {post} /api/nodes/{id} 8. Trigger an agent run
+    @apiVersion 8.0.0
+    @apiName applyPolicy 
+    @apiGroup Nodes
+
+    @apiUse nodeId
+    @apiParam (Multi) {class}       classes Set class passed as parameter to make it available during agent run.
+
+    @apiDescription This API allows to trigger an agent run on the target node. Response is not a json but the actual agent output on the node.
```
Comment (fanf) [inline src/api_v2/node.md]: This API allows to trigger an agent run on the target node. Response is the streamed agent output directly from the node (and so not in JSON).

**Comment by amousset** [comment]
@VinceMacBuche This needs to be re-targeted to 4.1.

**Comment by amousset** [comment]
ping @VinceMacBuche

**Comment by VinceMacBuche** [comment]
PR rebased

**Comment by VinceMacBuche** [comment]
Commit modified

File: src/api_v2/nodes.md [hunk 3]
```diff
@@ -877,6 +877,110 @@ HTTP/1.1 200 OK
...
+
+    /**
+    @api {post} /api/nodes/{id} 7. Trigger all agent run
+    @apiVersion 8.0.0
+    @apiName applyPolicyAllNodes 
+    @apiGroup Nodes
+
+    @apiParam (Multi) {class}       classes Set class passed as parameter to make it available during agent run.
+
+    @apiDescription This API allows to trigger an agent run on the target node. Response content a json stating if agent could be started on each node, but not if the run went fine and do not display any output from it. You cann see the result of the run in rudder web interface or in the each agent logs
```
Comment (amousset) [inline src/api_v2/nodes.md]: cann -> can

Comment (amousset) [inline src/api_v2/nodes.md]: rudder -> Rudder

Comment (amousset) [inline src/api_v2/nodes.md]: content -> contains

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**